tools/docker/builder/Dockerfile

```
@@ -23,5 +23,11 @@ RUN apt-get update && apt-get install -y \
 	vim \
     && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /root/prplMesh
-ENTRYPOINT ["/root/prplMesh/tools/maptools.py", "build"]
+ARG uid
+ARG user
+ARG topdir
+RUN useradd -d /home/$user -m -s /bin/bash $user -u $uid
+
+USER $user
+WORKDIR $topdir
+ENTRYPOINT ["./prplMesh/tools/maptools.py", "build"]
```

You don't need to specify WORKDIR since maptools handles relative paths:
```suggestion
ENTRYPOINT ["$topdir/prplMesh/tools/maptools.py", "build"]
```
Though I don't know if variable references work within strings, it may need to be
```suggestion
ENTRYPOINT [$topdir + "/prplMesh/tools/maptools.py", "build"]
```

---

ENTRYPOINT doesn't support arguments, so we'll leave it as is

---

> ENTRYPOINT doesn't support arguments, so we'll leave it as is

Huh? From the Dockerfile man page:

         -- ENTRYPOINT has two forms:

                # executable form
                ENTRYPOINT ["executable", "param1", "param2"]`

                # run command in a shell - /bin/sh -c
                ENTRYPOINT command param1 param2

---

Anyway not so important, so I'll merge as is.